errata/README.md

```diff
@@ -0,0 +1,61 @@
+
+## This repository
+
+This repository contains a simple file with some associated scripts to handle and document errata for published Recommendations. The "model" is as follows:
+
+* Errata are raised and managed in a GitHub repository (“errata repository” in what follows). This often means re-using the repository used by a Working Group, if one exists.  If one or more individuals wishes to use this model for a Recommendation which does not have a draft in the W3C GitHub repository, please send an inquiry to the [W3C webmaster](webreq@w3.org) so that they can create one for you.
+* There is a selected group of administrators (not only the team…) that can change the labels for issues in the repository.
+
+Note that the workflow describes the case whereby a single repository is used for several documents. In case the Working Group follows the separate repository per document model, an errata file must be installed separately for each document.
+
+## How does it work for the user
+
+Raising and managing errata is done as follows. Note that the approach below is for a case when the errata repository is used for several recommendations (identified by separate labels, `Model` and `Vocab` in the example below). Obviously, if a separate errata repository is used for separate documents, that approach can be simplified.
+
+* Errata are introduced and stored in the issue list of the dedicated repository. The workflow to add a new erratum is as follows:
+  * An issue is raised for a possible erratum. The label of the issue SHOULD be set to “`ErratumRaised`”. It SHOULD also include the label corresponding to the document on which the erratum is raised, e.g., “`Model`” or “`Vocab`”. It is o.k. for an erratum to have several labels. In some, exceptional, cases, i.e., when the erratum is very general, it is also acceptable not to have a reference to a document.
+  * The community discusses the issue. If it is accepted as a genuine erratum, the label “`Errata`” is added to the entry and the “`ErratumRaised`” label should be removed. Additionally, a new comment on the issue MAY be added, beginning with the word "Summary:" (if such a summary is useful based on the discussion).
+  * If the community rejects the issue as an erratum, the issue should be closed.
+  * Each errata may be labelled as “`Editorial`”; editorial errata are listed separately from the substantial ones. By default, an issue is considered substantive unless indicated otherwise.
+  * ALL substantive errata are generally expected to have corresponding test(s) (such as a pull request in [web-platforms-tests](https://github.com/web-platforms-tests/wpt)), either in the form of new tests or modifications to existing tests, or must include the rationale for why test updates are not required for the erratum.
```

Comment:
```suggestion
```

Reply:
Thanks, @riccardoAlbertoni . I kept this file just temporarily, but it is actually not worth keeping it, at least in its current form. 

I think we can drop all the content and replace it with just a statement saying that the errata page has been created based on etc.

Reply:
Just done.

Are you happy with this revision, @riccardoAlbertoni ?

Reply:
Fine to me.